Error 1: TestProducts.test_deserialize_a_product

**As a member of the products team** ...
**I need to** fix the errors in pytest
**So that** our code runs


**Assumptions:**
* Run pytest in vagrant

**Acceptence Criteria:**
...

Error no longer shows up
...

________________________________________________________ TestProducts.test_deserialize_a_product ________________________________________________________

self = <tests.test_products.TestProducts testMethod=test_deserialize_a_product>

    def test_deserialize_a_product(self):
        """ Test deserialization of a product """
        data = {"id": 1, "name": "T-Shirt", "category": "Clothing", "available": True, "Price": 35}
        product = Products()
>       product.deserialize(data)

tests/test_products.py:120: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <product 'T-Shirt'>, data = {'Price': 35, 'available': True, 'category': 'Clothing', 'id': 1, ...}

    def deserialize(self, data):
        """
        Deserializes a product from a dictionary
    
        Args:
            data (dict): A dictionary containing the product data
        """
        try:
            self.name = data['name']
            self.category = data['category']
            self.available = data['available']
            self.price = data['price']
        except KeyError as error:
>           raise DataValidationError('Invalid product: missing ' + error.args[0])
E           DataValidationError: Invalid product: missing price

app/models.py:92: DataValidationError

--- a/tests/test_products.py
+++ b/tests/test_products.py
@@ -88,7 +88,7 @@ class TestProducts(unittest.TestCase):
         # but the data did change
         products = Products.all()
         self.assertEqual(len(products), 1)
-        self.assertEqual(product[0].category, "Electronics")
+        self.assertEqual(product[1].category, "Electronics")
 
     def test_delete_a_product(self):
         """ Delete a product """
@@ -115,7 +115,7 @@ class TestProducts(unittest.TestCase):
 
     def test_deserialize_a_product(self):
         """ Test deserialization of a product """
-        data = {"id": 1, "name": "T-Shirt", "category": "Clothing", "available": True, "Price": 35}
+        data = {"id": 1, "name": "T-Shirt", "category": "Clothing", "available": True, "price": 35}
         product = Products()
         product.deserialize(data)
         self.assertNotEqual(product, None)

